network-api/networkapi/wagtailpages/pagemodels/buyersguide/utils.py

```diff
@@ -43,8 +43,12 @@ def sort_average(products):
     return sorted(products, key=lambda p: p.creepiness)
 
 
-def get_featured_cta(self):
+def get_bg_featured_cta(page):
```

Comment:
What do you think about spelling out `buyersguide` instead of using the abbreviation? Currently the abbreviation seems clear, because we are working on it. I wonder if this is true in the future.

Reply:
Makes sense to me! Updated 👍